content/about.md

```diff
@@ -3,15 +3,15 @@ title: About us
 image: images/profile.png
 ---
 ## About us
-DevRel Collective exists to facilitate sharing information, resources, and encouragement among the Developer Relations community.
+The DevRel Collective exists as a place for DevRel professionals, Community Managers, and others to share resources, learn best practices, support one another, and be amongst our peers. We participate in and nurture communities as our jobs, but we, too need a community of our own. This is that place. We are over **1,800 professionals** (and growing all the time!) from all walks of life and all over the world.
```

Comment:
We should make sure the info in [this pull request](https://github.com/devrelcollective/getting-started/pull/52/files) matches the new info here. I'm not picky about which version we go with, but it should be the same to reduce any possible confusion.